spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -89,6 +89,24 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  // based on Spark's SQLWindowFunctionSuite test of the same name
+  test("window function: partition and order expressions") {
+    for (shuffleMode <- Seq("auto", "native", "jvm")) {
```

Comment:
Does shuffle mode affect the behavior of window function?

Reply:
Not directly, but due to an existing issue in Comet, we fall back to Spark for if shuffle mode is auto or native, so we only see the issue when specifying jvm shuffle.

See https://github.com/apache/datafusion-comet/issues/1248 for more info.